src/webview/wkwebview/mod.rs

```diff
@@ -406,8 +407,9 @@ impl InnerWebView {
                   if !function.is_null() {
                     let function = &mut *(*function
                       as *mut (Box<dyn for<'r> Fn(&'r Window, String)>, Rc<Window>));
-                    let title = NSString(change.valueForKey_(key_path)).to_str().to_string();
-                    (function.0)(&function.1, title);
+                    let webview = this.get_ivar::<id>("webview");
+                    let title: id = msg_send![*webview, title];
```

Comment:
`_of_object` is the source object. You can just use it directly:
```
let title: id = msg_send![_of_object, title];
```
But probably rename to object is better.

Reply:
updated but kept the name as `of_object` as it seemed appropriate to keep close to the apple docs.